File: 0412.py
class Solution:
    def fizzBuzz(a,n)->List[str]:
        x=[]
        for i in range(1,n+1):
            b=""
            if i%3==0:b+='Fizz'
            if i%5==0:b+='Buzz'
            if b=='':b=str(i)
            x.append(b)
        return x

File: 0412.js
var fizzBuzz=function(n){var x=[];for(let i=1;i<=n;i++){i%15==0?x.push("FizzBuzz"):i%5==0?x.push("Buzz"):i%3==0?x.push("Fizz"):x.push(i+"")}return x}
